packages/execution-environments/src/common/sortParams.ts

```diff
@@ -2,6 +2,15 @@
 
 import { JSONRPCParams } from '../__GENERATED__/openrpc';
 
+/**
+ * Deterministically sort JSON-RPC parameter keys.
```

Comment:
This is done to support both arrays and objects as parameters. But turns both of these into arrays for easier consumption by the snap.